### PR TITLE
Make it compile

### PR DESCRIPTION
**Pull request by markusthoemmes**

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -3,13 +3,12 @@ module knative.dev/kn-plugin-source-kamelet
 go 1.15
 
 require (
-	github.com/apache/camel-k v1.3.1
```
Comment (markusthoemmes) [inline go.mod:6]: This pulls in half of the world so try not to depend on it.

File: go.mod [hunk 2]
```diff
@@ -3,13 +3,12 @@ module knative.dev/kn-plugin-source-kamelet
...
 	github.com/apache/camel-k/pkg/client/camel v1.3.1
 	github.com/spf13/cobra v1.1.3
 	gotest.tools v2.2.0+incompatible
-	k8s.io/apimachinery v0.18.12
-	k8s.io/client-go v12.0.0+incompatible
-	knative.dev/client v0.20.0
-	knative.dev/hack v0.0.0-20210120165453-8d623a0af457
+	k8s.io/apimachinery v0.19.7
+	k8s.io/client-go v0.19.7
+	knative.dev/client v0.21.0
```
Comment (markusthoemmes) [inline go.mod:12]: Updated this to align on 0.19 deps and it's the most recent anyway.

File: internal/command/types.go [hunk 3]
```diff
@@ -41,7 +42,8 @@ func (params *KameletPluginParams) Initialize() {
 }
 
 func (params *KameletPluginParams) newKameletClient() (camelkv1alpha1.CamelV1alpha1Interface, error) {
-	client, err := camelk.NewClient(true)
+	// TODO: This needs the Rest config plumbed through or alternatively look it up afresh.
+	client, err := camelk.NewForConfig(nil)
```
Comment (markusthoemmes) [inline internal/command/types.go:46]: This is essentially the "raw" client-go generated client. I'd hope that the kn plugin arch allows for plumbing through the rest config, if not you can look it up yourself.

**Comment by christophd** [comment]
@markusthoemmes thanks a lot

**Pull request merged**